NitroxLauncher/App.xaml.cs

```diff
@@ -30,6 +30,31 @@ protected override void OnStartup(StartupEventArgs e)
             {
                 MessageBox.Show("Nitrox launcher should not be executed from a temporary directory. Install Nitrox launcher properly by extracting ALL files and moving these to a dedicated location on your PC.", "Invalid working directory", MessageBoxButton.OK, MessageBoxImage.Error);
                 Environment.Exit(1);
+                Shutdown();
```

Comment:
Environment.Exit will kill the app before this call so `Shutdown()` is unreachable.

Either remove the `Environment.Exit(1)` or remove `Shutdown()`.